src/transport/mod.rs

```diff
@@ -175,6 +175,23 @@ bitflags! {
     }
 }
 
+/// ISR status field. Reading this register resets it to 0 and causes the device
+/// to de-assert the interrupt.
```

Comment:
I think this comment belongs on the field rather than the type. This type is part of the public API of all the drivers, so it makes more sense to describe it from that point of view. E.g. something like "The set of interrupts which were pending".